src/core/client.js

```diff
@@ -142,6 +143,8 @@ export class Client {
      * @private
      */
     this.activeUserStorage = new ActiveUserStorage();
+
+    this.storageType = config.storageType || storageType.inmemory;
```

Comment:
We just made some changes related to https://kinvey.atlassian.net/browse/MLIBZ-2120 that uses `config.storage` to set the type of storage to use. Is this the same thing that this is doing above? If it is, shouldn't we keep the names the same?

Reply:
This predates the feature by some time. It will have to conform to the already released API, so a slight change is necessary. I haven't put in too much effort into the implementation, since it wasn't planned to be a big part of the redesign. I just felt we better integrate a basic version of it it early on, to avoid double work later. The specifics will be ironed out now that we've committed to a public API.

Reply:
Ok

Reply:
This will be fixed with https://kinvey.atlassian.net/browse/MLIBZ-2329.